bill/invoice.go

```diff
@@ -95,6 +95,9 @@ type Invoice struct {
 	// Additional complementary objects that add relevant information to the invoice.
 	Complements []*schema.Object `json:"complements,omitempty" jsonschema:"title=Complements"`
 
+	// Extension code map for any additional regime specific codes that may be required.
+	Ext cbc.CodeMap `json:"ext,omitempty" jsonschema:"title=Ext"`
```

Comment:
I couldn't find a good place to store Mabe's `reference1` and `reference2` fields so I added this Ext codeMap at invoice level. The content of `reference1` is a bit ambiguous, and changes depending on the case:

![image](https://github.com/invopop/gobl/assets/856/585d532b-49f6-4c5b-b5db-e4f3bc2f39ef)

In the case of `reference2` this is what the doc says: "Referencia adicional para uso futuro o bien "NA" o "0"".

Suggestions on a better place to put this data are welcome!

Reply:
See the `addenda-mabe.yaml` below to see it in context